app/src/main/java/com/sdp/movemeet/backend/FirebaseInteraction.java

```diff
@@ -41,39 +41,6 @@
 
     private static final String TAG = "FirebaseInteraction";
 
-    /**
-     * Retrieve user data (full name, email, phone number and profile description) from Firebase
-     * Firestore and directly update corresponding TextViews.
-     * @param fStore Firebase Firestore reference
-     * @param userId The ID of the user
-     * @param textViewArray A TextView array containing the TextViews referencing user data (full
-     *                      name, email, phone number and profile description) to be updated
-     * @param activity The activity from which this function is called
-     * @return An array of TextViews containing the updated user data
-     */
-    public static TextView[] retrieveDataFromFirebase(FirebaseFirestore fStore, String userId, TextView[] textViewArray, Activity activity) {
-        DocumentReference documentReference = fStore.collection("users").document(userId);
-        documentReference.addSnapshotListener(activity, new EventListener<DocumentSnapshot>() {
-            @Override
-            public void onEvent(@Nullable DocumentSnapshot documentSnapshot, @Nullable FirebaseFirestoreException e) {
-                if (documentSnapshot == null) {
-                    //nothing
-                } else if (textViewArray.length == 4) {
-                    textViewArray[0].setText(documentSnapshot.getString("fullName"));
-                    textViewArray[1].setText(documentSnapshot.getString("email"));
-                    textViewArray[2].setText(documentSnapshot.getString("phone"));
-                    textViewArray[3].setText(documentSnapshot.getString("description"));
-                } else {
-                    textViewArray[0].setText(documentSnapshot.getString("fullName"));
-                    textViewArray[1].setText(documentSnapshot.getString("email"));
-                    textViewArray[2].setText(documentSnapshot.getString("phone"));
-                }
-            }
-        });
-
-        return textViewArray;
-    }
-
```

Comment:
Hoping no one is using this method anymore 🤞

Reply:
Haha true ! Rest in peace